sdk/src/types/block/address/restricted.rs

```diff
@@ -54,6 +55,8 @@ impl RestrictedAddress {
     }
 }
 
+impl StorageScore for RestrictedAddress {}
```

Comment:
I know iota.go returns 0 and that it's currently correct but I'd personally forward to the address, less error-prone. Just resolve if you think it's not worth it.